modules/internal/OwnedObject.chpl

```diff
@@ -225,8 +207,9 @@ module OwnedObject {
        of the argument. The result has the same type as the argument.
        If the argument is non-nilable, it must be recognized by the compiler
        as an expiring value. */
+    @deprecated(notes="owned.create is deprecated - please use :proc:`owned.adopt` instead")
```

Comment:
Stylistically, I'd throw quotes around 'owned.create' and/or writing it as 'owned.create()' (with parens), which might suggest doing the same for 'owned.adopt' as well.  Essentially, I try to set all code words that might potentially be mistaken for English word apart with single quotes like this (though if they have `()`, I'm open to that being enough of a visual cue that these are not English words to not require it).  I'm not sure how that interacts with the `:proc:` hyperlink, though, nor how closely or different it is from other deprecation warnings we currently have.

Reply:
Now seeing everything that would have to change if this did, let's not do this tonight / in this PR.